packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/AddBankYapily/Connect/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+import { bindActionCreators, Dispatch } from 'redux'
+
+import { Remote } from 'blockchain-wallet-v4/src'
+import DataError from 'components/DataError'
+import { WalletFiatType } from 'core/types'
+import { actions, selectors } from 'data'
+import { RootState } from 'data/rootReducer'
+
+import { Loading, LoadingTextEnum } from '../../../../components'
```

Comment:
I'm not a big fan of relative imports like we might change it to be real relative import without `../../..`

Reply:
I don't love it either. We have so many different places were we define reusable components that I ended up with imports like:

```
import { foo } from './components'
import { bar } from '../../components'
import { baz } from '../components'
```

So instead of rearchitecting the entire system I put them all in the top most components dir for now. Let's chat IRL about how we can come up with a better system.